src/oic/utils/keyio.py

```diff
@@ -777,6 +789,39 @@ def get_issuer_keys(self, issuer):
             res.extend(kbl.keys())
         return res
 
+    def __eq__(self, other):
+        if not isinstance(other, KeyJar):
+            return False
+
+        # The set of issuers MUST be the same
+        if set(self.keys()) != set(other.keys()):
+            return False
+
+        # Keys per issuer must be the same
+        for iss in self.keys():
+            sk = self.get_issuer_keys(iss)
+            ok = other.get_issuer_keys(iss)
+            if len(sk) != len(ok):
```

Comment:
The comment above suggests the key *values* must be the same, but only length or?